src/i18n/en.js

```diff
@@ -449,42 +449,43 @@ You'll find an Email with steps to follow.`,
     go_foward_pages: 'Go forward 5 pages',
   },
   plan_calculator: {
-    banner_for_monthly_deliveries: `Looking for something you cannot find? We recommend you to try with our Custom Made Plans. <Link>CONTACT US</Link> and we'll help you.`,
-    banner_for_prepaid: `Do you have more than 100,000 Subscribers? We recommend you to see our <Link>HIGH VOLUME PLANS</Link>.`,
-    banner_for_subscribers: `Do you have more than 100,000 Subscribers? We recommend you to see our <Link>HIGH VOLUME PLANS</Link>.`,
+    banner_for_monthly_deliveries: `Do you need more Emails? <Link>Write to us</Link>.`,
+    banner_for_prepaid: `Do you need more Credits? We advise you to hire a <Link>Plan by Emails</Link>.`,
```

Comment:
```suggestion
    banner_for_prepaid: `Do you need more Credits? We reccomend you to hire a <Link>Plan by Emails</Link>.`,
```
How about this @jmonacchi ? https://linguix.com/blog/advise-recommend-suggest-which-verb-to-choose/#:~:text=Advise%20is%20a%20more%20formal,based%20on%20your%20own%20experience.

Reply:
Like both of them! Let's use recommend (correct spelling) and get. I will update them on Confluence pages too.